test/residency/contract/test_catalog_public_seam.py

```
@@ -0,0 +1,153 @@
+            "-o",
+            str(output),
+        ]
+    )
+    return command
+
+
+def require_posix_command_contract() -> None:
+    output = Path("catalog-public-seam")
+    fallback = compiler_command(output, configured=["c++"], environment={})
+    if "-lmbedcrypto" not in fallback:
+        raise AssertionError("POSIX catalog seam omitted the mbedcrypto fallback")
```

Ruff (0.16.1)

[warning] Avoid specifying long messages outside the exception class

(TRY003)